source/material_model/steinberger.cc

```diff
@@ -189,22 +189,24 @@ namespace aspect
         delta_temperature = temperature-adiabatic_temperature;
 
       // For an explanation on this formula see the Steinberger & Calderwood 2006 paper
-      // We here compute the lateral variation of viscosity due to temperature (vis_lateral) as
+      // We here compute the lateral variation of viscosity due to temperature (temperature_prefactor) as
       // V_lT = exp [-(H/nR)*dT/(T_adiabatic*(T_adiabatic + dT))] as in Eq. 6 of the paper.
       // We get H/nR from the lateral_viscosity_lookup->lateral_viscosity function.
-      const double vis_lateral_exp = -1.0*lateral_viscosity_lookup->lateral_viscosity(depth)*delta_temperature/(temperature*adiabatic_temperature);
+      const double log_temperature_prefactor = -1.0*lateral_viscosity_lookup->lateral_viscosity(depth)*delta_temperature/(temperature*adiabatic_temperature);
 
       // Limit the lateral viscosity variation to a reasonable interval
-      const double vis_lateral = std::max(std::min(std::exp(vis_lateral_exp),max_lateral_eta_variation),1/max_lateral_eta_variation);
+      const double temperature_prefactor = std::max(std::min(std::exp(log_temperature_prefactor),max_lateral_eta_variation),1/max_lateral_eta_variation);
 
       //Visc_rT = exp[(H/nR)/T_adiabatic], Eq. 7 of the paper
-      const double vis_radial = radial_viscosity_lookup->radial_viscosity(depth);
+      const double eta_ref = radial_viscosity_lookup->radial_viscosity(depth);
 
-
-      const double vis_compositional = MaterialUtilities::average_value (volume_fractions, viscosity_prefactors, viscosity_averaging);
+// std::cout<<"size vf"<<volume_fractions.size()<<std::endl;
+// std::cout<<"size vis_pre"<<viscosity_prefactors.size()<<std::endl;
```

Comment:
You added some lines again here that are code you commented out and that is no longer needed. Please remove them.